contrib/lxqt-panel/template.py

```diff
@@ -0,0 +1,38 @@
+pkgname = "lxqt-panel"
+pkgver = "2.0.1"
+pkgrel = 0
+build_style = "cmake"
+configure_args = [
+    "-DCPULOAD_PLUGIN=NO",
+    "-DNETWORKMONITOR_PLUGIN=NO",
+    "-DSYSSTAT_PLUGIN=NO",
```

Comment:
any reason to disable these

Reply:
The reason is that I was too lazy to package the required dependencies